src/DataWrangling/metadata.jl

```diff
@@ -2,10 +2,25 @@ using CFTime
 using Dates
 using Base: @propagate_inbounds
 
-struct Metadata{V, D}
+struct BoundingBox{X, Y, Z}
+    longitude :: X
+    latitude :: Y 
+    z :: Z
+end
+
+"""
+    BoundingBox(; longitude=nothing, latitude=nothing, z=nothing)
+
+Create a bounding box with `latitude`, `longitude`, and `z` bounds on the sphere.
+"""
+BoundingBox(; longitude=nothing, latitude=nothing, z=nothing) =
```

Comment:
I think this is a great way to unify the JRA55 field time series with the already existing general field time series method we have for metadata, where the only difference with the other datasets is that with JRA55, we also provide a latitude and longitude bound (we can unify everything in another PR).

Reply:
We need bounding box pretty generally. Also for ERA5 atmosphere

Reply:
I was debating making it `LatitudeLongitudeBoundingBox`, but I think that almost all datasets will be lat-lon (I don't see a use for Cartesian geometry bounding boxes... yet)